Makefile

```diff
@@ -700,6 +702,20 @@ generate-images:
 	npm install --no-save --no-package-lock fabric imagemin-zopfli
 	node build/generate-images.js $(TAGS)
 
+.PHONY: generate-manpage
+generate-manpage:
+	@[ -f gitea ] || make backend
+	@mkdir -p man/man1/ man/man5
+	@./gitea docs --man > man/man1/gitea.1
+	@gzip -9 man/man1/gitea.1 && echo man/man1/gitea.1.gz created
+	@#TODO A smal script witch format config-cheat-sheet.en-us.md nicely to suit as man page
+	@tail -n +36 docs/content/doc/advanced/config-cheat-sheet.en-us.md > man/man5/gitea.app.ini.5.md
```

Comment:
I don't know that I like this part. The config cheat sheet changes [fairly often](https://github.com/go-gitea/gitea/commits/master/docs/content/doc/advanced/config-cheat-sheet.en-us.md) and an arbitrary `+36` seems prone to breaking.  
To be fair, the changes are usually in the config parts, which are after line 36, however any changes to the beginning paragraphs would potentially break this.

As a first PR maybe the docs man pages are enough? Can try to get something more precise to add the config later?